src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -91,6 +92,12 @@ public void parseCommand_list() throws Exception {
         assertTrue(parser.parseCommand(ListCommand.COMMAND_WORD + " 3") instanceof ListCommand);
     }
 
+    @Test
+    public void parseCommand_listGroup() throws Exception {
+        assertTrue(parser.parseCommand(ListGroupCommand.COMMAND_WORD) instanceof ListGroupCommand);
+        assertTrue(parser.parseCommand(ListGroupCommand.COMMAND_WORD + " 3") instanceof ListGroupCommand);
```

Comment:
Hmm this is quite interesting, so if I key `help 3`/`list-group 3`, the command still works - is this a feature or a bug?

We can consider checking for this in another issue.